packages/x-license-pro/src/verifyLicense/verifyLicense.test.ts

```diff
@@ -8,36 +8,98 @@ const oneYear = oneDayInMS * 365;
 const RELEASE_INFO = generateReleaseInfo();
 
 describe('License: verifyLicense', () => {
-  const validLicense = generateLicense({
-    expiryDate: new Date(new Date().getTime() + oneYear),
-    orderNumber: 'MUI-123',
-  });
+  describe('key version: 1', () => {
+    const license =
+      '0f94d8b65161817ca5d7f7af8ac2f042T1JERVI6TVVJLVN0b3J5Ym9vayxFWFBJUlk9MTY1NDg1ODc1MzU1MCxLRVlWRVJTSU9OPTE=';
```

Comment:
We don't have the v1 generation anymore but it's worth testing it
This key is the doc key